remoteconfig_test.go

```diff
@@ -616,7 +616,8 @@ func (s *RemoteConfigSuite) TestReadJSONValidateEmbeddedStruct() {
 
 	c := &SampleConfig{}
 	err := ReadJSONValidate(cfgBuffer, c)
-	assert.EqualError(s.T(), err, "Failed to decode JSON, with error, json: cannot unmarshal string into Go value of type int64")
+	assert.NotNil(s.T(), err)
```

Comment:
There's an `assert.Error` that provide a slightly more obvious error message than you get with `assert.NotNil`.